node/src/Commands.ts

```diff
@@ -2712,9 +2751,14 @@ export function createGeoSearch(
     }
 
     if (resultOptions) {
-        if (resultOptions.withCoord) args.push("WITHCOORD");
-        if (resultOptions.withDist) args.push("WITHDIST");
-        if (resultOptions.withHash) args.push("WITHHASH");
+        if ("withCoord" in resultOptions && resultOptions.withCoord)
```

Comment:
Is the first check needed? Would it be false even if it doesn't exist?

Reply:
First check is needed to confirm that I can access to `with*`. I have to update this code, because there are 2 derived types with different sets of fields.